doc/adr/0006-no-certificate-pinning.md

```diff
@@ -25,4 +25,4 @@ known on start of the identification flow.
 ## Consequences
 
 A man-in-the-middle attack on the TLS connection is possible if an attacker comes into possession of a 
-certificate signed by a trusted root CA for a domain used in the identification flow.
\ No newline at end of file
+certificate signed by a trusted root CA for a domain used in the identification flow.
```

Comment:
What was changed in this line? 🤔

Reply:
Nothing. I don't know why this is in. Maybe because I edited the filename.

Reply:
Sometimes it's about having a newline at the end or not. Maybe that's what changed.